earthfile2llb/earthfile2llb.go

```diff
@@ -40,6 +40,8 @@ type ConvertOpt struct {
 	VarCollection *variables.Collection
 	// A cache for image solves. depTargetInputHash -> context containing image.tar.
 	SolveCache map[string]llb.State
+	// WithSSH will pass along the ssh auth agent socket when true
+	WithSSH bool
```

Comment:
This is no longer necessary, right?

Reply:
good eyes, I missed that one. it's gone now.